[0.009s] Invoking command in '/home/user/ros2_control_example/build/custom_interfaces': CMAKE_PREFIX_PATH=/opt/ros/humble /usr/bin/cmake /home/user/ros2_control_example/src/custom_interfaces -DCMAKE_INSTALL_PREFIX=/home/user/ros2_control_example/install/custom_interfaces
[0.017s] -- Found ament_cmake: 1.3.3 (/opt/ros/humble/share/ament_cmake/cmake)
[0.120s] -- Found std_msgs: 4.2.2 (/opt/ros/humble/share/std_msgs/cmake)
[0.138s] -- Found rosidl_generator_c: 3.1.4 (/opt/ros/humble/share/rosidl_generator_c/cmake)
[0.141s] -- Found rosidl_adapter: 3.1.4 (/opt/ros/humble/share/rosidl_adapter/cmake)
[0.145s] -- Found rosidl_generator_cpp: 3.1.4 (/opt/ros/humble/share/rosidl_generator_cpp/cmake)
[0.154s] -- Using all available rosidl_typesupport_c: rosidl_typesupport_fastrtps_c;rosidl_typesupport_introspection_c
[0.166s] -- Using all available rosidl_typesupport_cpp: rosidl_typesupport_fastrtps_cpp;rosidl_typesupport_introspection_cpp
[0.208s] -- Found rosidl_default_generators: 1.2.0 (/opt/ros/humble/share/rosidl_default_generators/cmake)
[0.329s] [31mCMake Error at /opt/ros/humble/share/rosidl_adapter/cmake/rosidl_adapt_interfaces.cmake:59 (message):
[0.329s]   execute_process(/usr/bin/python3.10 -m rosidl_adapter --package-name
[0.329s]   custom_interfaces --arguments-file
[0.330s]   /home/user/ros2_control_example/build/custom_interfaces/rosidl_adapter__arguments__custom_interfaces.json
[0.330s]   --output-dir
[0.330s]   /home/user/ros2_control_example/build/custom_interfaces/rosidl_adapter/custom_interfaces
[0.330s]   --output-file
[0.330s]   /home/user/ros2_control_example/build/custom_interfaces/rosidl_adapter/custom_interfaces.idls)
[0.330s]   returned error code 1:
[0.330s] 
[0.330s]   Error processing ' builtin_interfaces/Time stamp' of
[0.330s]   'custom_interfaces/Controlnote': 'string index out of range'
[0.330s] 
[0.330s]   Traceback (most recent call last):
[0.330s] 
[0.330s]     File "/usr/lib/python3.10/runpy.py", line 196, in _run_module_as_main
[0.330s]       return _run_code(code, main_globals, None,
[0.330s]     File "/usr/lib/python3.10/runpy.py", line 86, in _run_code
[0.330s]       exec(code, run_globals)
[0.330s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/__main__.py", line 19, in <module>
[0.330s]       sys.exit(main())
[0.330s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/main.py", line 53, in main
[0.331s]       abs_idl_file = convert_to_idl(
[0.331s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/__init__.py", line 19, in convert_to_idl
[0.331s]       return convert_msg_to_idl(
[0.331s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/msg/__init__.py", line 28, in convert_msg_to_idl
[0.331s]       msg = parse_message_string(package_name, input_file.stem, content)
[0.331s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/parser.py", line 520, in parse_message_string
[0.331s]       Type(type_string, context_package_name=pkg_name),
[0.332s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/parser.py", line 241, in __init__
[0.332s]       self.is_array = type_string[-1] == ']'
[0.332s] 
[0.332s]   IndexError: string index out of range
[0.332s] 
[0.332s] Call Stack (most recent call first):
[0.332s]   /opt/ros/humble/share/rosidl_cmake/cmake/rosidl_generate_interfaces.cmake:130 (rosidl_adapt_interfaces)
[0.332s]   CMakeLists.txt:17 (rosidl_generate_interfaces)
[0.332s] 
[0.332s] [0m
[0.332s] -- Configuring incomplete, errors occurred!
[0.332s] See also "/home/user/ros2_control_example/build/custom_interfaces/CMakeFiles/CMakeOutput.log".
[0.334s] Invoked command in '/home/user/ros2_control_example/build/custom_interfaces' returned '1': CMAKE_PREFIX_PATH=/opt/ros/humble /usr/bin/cmake /home/user/ros2_control_example/src/custom_interfaces -DCMAKE_INSTALL_PREFIX=/home/user/ros2_control_example/install/custom_interfaces
